python/python/res/simulator/robust_simulator.py

```diff
@@ -0,0 +1,121 @@
+from ecl.util import BoolVector
+
+from res.enkf import ErtRunContext, EnKFMain, EnkfConfigNode, EnkfNode, NodeId
+from res.server import SimulationContext
+from .robust_simulator_context import RobustContext
+
+class RobustSimulator(object):
```

Comment:
Why is it called `RobustSimulator`? Would `BatchSimulator` be a better name or is this really tightly connected to doing robust optimization?

Reply:
I have renamed